content/what-is-passwordless-authentication/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "What is Passwordless Authentication: The Cool Kid on The Block? 🔒🚫"
+# date: "2024-07-07"
```

Comment:
You can set dummy values for this. We have a netlify build which allows us to preview the blogs in the PR itself. These values need to be defined for that build to run

Reply:
Thank you!

Reply:
Cover is not showing up properly on the netlify build. Will look why now

Reply:
Cool, works with the blog. Now resizing it to add to the card covers.